backend/src/database/migrations/20221130131614-create-notification-data-by-campaign-id-function.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    queryInterface.createFunction(
+      'get_notification_data_by_campaign_id',
```

Comment:
hmm why do we need to put this in a sub-routine? Can't we just write raw query inside the code?

Reply:
this is a personal implementation choice, as raw queries poses severe security vulnerabilities and extremely high chances of error. unless i can use ORM, this is the next safest method. to be refactored once models are made available to worker